addons/mail/static/src/models/message_tracking_value.js

```diff
@@ -0,0 +1,182 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+import { format } from 'web.field_utils';
+
+registerModel({
+    name: 'MessageTrackingValue',
+    identifyingFields: ['id'],
+    modelMethods: {
+        //----------------------------------------------------------------------
+        // Public
+        //----------------------------------------------------------------------
+        /**
+         * @static
+         * @param {Object} data
+         * @return {Object}
+         */
+        convertData(data) {
+            const data2 = {};
+            if ('id' in data) {
+                data2.id = data.id;
+            }
+            if ('changed_field' in data) {
+                data2.changedField = data.changed_field;
+            }
+            if ('currency_id' in data) {
+                data2.currencyId = data.currency_id;
+            }
+            if ('field_type' in data) {
+                data2.fieldType = data.field_type;
+            }
+            if ('new_value' in data) {
+                data2.newValue = data.new_value;
+            }
+            if ('old_value' in data) {
+                data2.oldValue = data.old_value;
+            }
+            return data2;
+        },
+
+    },
+    recordMethods: {
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeChangedFieldAsString() {
+            return _.str.sprintf(this.env._t("%s:"), this.changedField);
+        },
```

Comment:
This was how it worked before. Should we make changes to this?

Reply:
I don't think we should remove translated term in general, and if we do because there is a reason here, I think that's unrelated to current PR so I would say no change.

Reply:
There are some translations on that. for example, msgid "Stage" -> msgstr "Faza"; msgid "Stage Changed" -> msgstr "Faza promjenjena". Maybe keeping the translation if better.